src/ProjectDirectory.php

```diff
@@ -6,6 +6,11 @@ class ProjectDirectory
 {
     public static function find(/* string */ $cwd)
     {
+        $env = getenv('WHIPPET_PROJECT_DIRECTORY');
+        if ($env) {
+            return \Result\Result::ok(new self($env));
```

Comment:
In the normal case we check for whippet.json - do we not care if it doesn't exist?